test/integration/document_collection_test.rb

```diff
@@ -9,11 +9,33 @@ class DocumentCollectionTest < ActionDispatch::IntegrationTest
 
   test "renders metadata and document footer" do
     setup_and_visit_content_item('document_collection')
-    assert_has_component_metadata_pair("first_published", "29 February 2016")
+    assert_has_publisher_metadata(
+      published: "Published 29 February 2016",
+      metadata: { "From:":
+        {
+          text: "Driver and Vehicle Standards Agency",
+          href: "/government/organisations/driver-and-vehicle-standards-agency"
+        } }
+    )
+    assert_footer_has_published_dates("Published 29 February 2016")
+  end
 
-    from = ["<a href=\"/government/organisations/driver-and-vehicle-standards-agency\">Driver and Vehicle Standards Agency</a>"]
-    assert_has_component_metadata_pair("from", from)
-    assert_has_component_document_footer_pair("from", from)
+  test "renders side bar navigation" do
```

Comment:
Can we rename this to 'renders related navigation sidebar', as we have a few different navigation sidebars